multichain-testing/test/deposit-withdraw-lca.test.ts

```diff
@@ -0,0 +1,222 @@
+import anyTest from '@endo/ses-ava/prepare-endo.js';
+import type { TestFn } from 'ava';
+import { AmountMath } from '@agoric/ertp';
+import { makeDoOffer } from '../tools/e2e-tools.js';
+import { makeQueryClient } from '../tools/query.js';
+import { commonSetup, SetupContextWithWallets } from './support.js';
+
+const test = anyTest as TestFn<SetupContextWithWallets>;
+
+const accounts = ['user1'];
+
+const contractName = 'basicFlows';
+const contractBuilder =
+  '../packages/builders/scripts/orchestration/init-basic-flows.js';
+
+test.before(async t => {
+  const { deleteTestKeys, setupTestKeys, ...rest } = await commonSetup(t);
+  deleteTestKeys(accounts).catch();
+  const wallets = await setupTestKeys(accounts);
+  t.context = { ...rest, wallets, deleteTestKeys };
+  const { startContract } = rest;
+  await startContract(contractName, contractBuilder);
+});
+
+test.after(async t => {
+  const { deleteTestKeys } = t.context;
+  deleteTestKeys(accounts);
+});
+
+test('Deposit IST to orchAccount and then withdraw', async t => {
+  const {
+    wallets,
+    provisionSmartWallet,
+    vstorageClient,
+    retryUntilCondition,
+    useChain,
+  } = t.context;
+
+  // Provision the Agoric smart wallet
+  const agoricAddr = wallets.user1;
+  const wdUser = await provisionSmartWallet(agoricAddr, {
+    BLD: 100n,
+    IST: 1000n,
+  });
+  t.log(`Provisioned Agoric smart wallet for ${agoricAddr}`);
+
+  const doOffer = makeDoOffer(wdUser);
+
+  // Create orchAccount
+  const makeAccountOfferId = `makeAccount-${Date.now()}`;
+  await doOffer({
+    id: makeAccountOfferId,
+    invitationSpec: {
+      source: 'agoricContract',
+      instancePath: [contractName],
+      callPipe: [['makeOrchAccountInvitation']],
+    },
+    offerArgs: { chainName: 'agoric' },
+    proposal: {},
+  });
+
+  // Wait for the orchAccount to be created
+  const { offerToPublicSubscriberPaths } = await retryUntilCondition(
+    () => vstorageClient.queryData(`published.wallet.${agoricAddr}.current`),
+    ({ offerToPublicSubscriberPaths }) =>
+      Object.fromEntries(offerToPublicSubscriberPaths)[makeAccountOfferId],
+    'makeAccount offer result is in vstorage',
+  );
+
+  const accountStoragePath = Object.fromEntries(offerToPublicSubscriberPaths)[
+    makeAccountOfferId
+  ]?.account;
```

Comment:
I'm surprised this optional doesn't cause a problem for the `.split` on 73. Maybe `offerToPublicSubscriberPaths` is `any? Not a blocker but consider typing.
```suggestion
  ]!.account;
```

Reply:
Yes, `offerToPublicSubscriberPaths` is `any`. Typing that here doesn't feel right - I've added a TODO for https://github.com/Agoric/agoric-sdk/issues/10214